mtags/src/main/scala-3/scala/meta/internal/pc/AutoImports.scala

```diff
@@ -183,6 +210,8 @@ object AutoImports:
           case AutoImport.SpecifiedOwner(sym) => importName(sym.owner)
           case AutoImport.Renamed(sym, rename) =>
             s"${importName(sym.owner.owner)}.{${sym.owner.nameBackticked} => $rename}"
+          case AutoImport.SelfRenamed(sym, rename) =>
+            s"${importName(sym.owner)}.{${sym.nameBackticked} => $rename}"
```

Comment:
Shouldn't it be unreachable?

Reply:
I don't think so, `SelfRenamed` will be caught here https://github.com/tanishiking/metals/blob/9cef97a78e0811aa7ff217a3d5ff1d8bbdd44f79/mtags/src/main/scala-3/scala/meta/internal/pc/AutoImports.scala#L178-L179 and then we will hit here in `importEdit`

Reply:
oh. now I see that SelfRenamed is a bit different from what I expected))
It's kind of auto import for `import` :thinking: